rocket/rocket.py

```diff
@@ -54,49 +53,53 @@ def launch(
             project_location: str = ".",
             dbfs_path: Optional[str] = None,
             watch: bool = True,
-            glob_path: Optional[Union[str, List[str]]] = None
+            glob_path: Optional[Union[str, List[str]]] = None,
+            use_volumes: Optional[bool] = False,
+            dst_path: Optional[str] = None,
```

Comment:
Maybe we deprecate `dbfs_path` since you can cover the use case with `dst_path`

Reply:
I added a warning there so people can still use it now but we can deprecate it in the next version.